backend/fms_core/viewsets/sample_next_step.py

```diff
@@ -0,0 +1,90 @@
+from rest_framework import viewsets
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.decorators import action
+
+from fms_core.models import SampleNextStep, StepSpecification
+from fms_core.serializers import SampleNextStepSerializer, StepSpecificationSerializer
+
+class SampleNextStepViewSet(viewsets.ModelViewSet):
+    queryset = SampleNextStep.objects.all()
+    serializer_class = SampleNextStepSerializer
```

Comment:
The serializer may not contain enough information for the frontend to build the lab work filtered list (2nd page in the lab work menu hierarchy). You would need to include the step and step specification in each serialized entry. We could also build a step/step_order endpoint with these details and stitch them in the frontend but we need them somehow.